package.json

```diff
@@ -265,20 +277,97 @@
           "group": "d2"
         }
       ]
-    },
-    "themes": [
+    }
+  },
```

Comment:
this is a duplicate entry of below, I don't think you needed to manually resolve merge conflicts if you ran the prettier lint command on this.